bokeh/application/handlers/script.py

```diff
@@ -1,6 +1,7 @@
 from __future__ import absolute_import, print_function
 
 import os
+import sys
```

Comment:
This is a bit worrying. Didn't we test imports? Or maybe we only test for unused imports?